external-crates/move/crates/move-compiler/src/command_line/compiler.rs

```diff
@@ -96,21 +114,44 @@ enum PassResult {
     Typing(typing::ast::Program),
     HLIR(hlir::ast::Program),
     CFGIR(cfgir::ast::Program),
-    Compilation(Vec<AnnotatedCompiledUnit>, /* warnings */ Diagnostics),
+    Compilation(
+        Vec<AnnotatedCompiledUnit>,
+        Vec<ModuleIdent>,
```

Comment:
Bit confused here, but I see one of two things happening, either 
A) Can we just call `AnnotatedCompiledUnit::module_ident`?
B) If we can't, can we add the ident to `AnnotatedCompiledUnit`?

Reply:
Happy with (B) but normally I try to avoid structural changes to the AST etc. if I can avoid it